devtools/server/actors/replay/telemetry.js

```diff
@@ -16,7 +32,8 @@ function pingTelemetry(source, name, data) {
       build: Services.appinfo.appBuildID,
       ts: Date.now(),
       source,
-      name
+      name,
+      browserUserToken: user,
     })
```

Comment:
I realize we could also instead do
```
const token = ReplayAuth.getOriginalApiKey() || ReplayAuth.getReplayUserToken();
```
and then pass the token value as a 
```
Authorization: `Bearer ${token}`
```
header like we do for all our other API calls, might be more consistent? That also leaves us more flexible on the backend because we can convert API keys back to their associated user if we want to.